src/i-shared-minter-mapping.ts

```diff
@@ -349,7 +349,7 @@ function tryLoadProject(
   return Project.load(fullProjectId);
 }
 
-function loadOrCreateMinterProjectAndConfigIfProject(
+export function loadOrCreateMinterProjectAndConfigIfProject(
```

Comment:
this could go in a common helpers file, but kind of liked keeping it in this file, as it is intended to be used for the new shared minters 🤷